README.md

```diff
@@ -63,18 +63,18 @@ Note you must have the option PROMPT_SUBST set, see zshoptions(1).
 prezto
 ---
 If you use [prezto](https://github.com/sorin-ionescu/prezto) you should do the following:
-
 ```
-cd ~/.zprezto/ \
-&& git submodule add https://github.com/miekg/lean.git modules/prompt/external/lean 2>/dev/null \
-&& git submodule update --init --recursive \
-&& cd modules/prompt/functions \
-&& ln -s ../external/lean/prompt_lean_setup
+cd $ZPREZTODIR && git clone --recurse-submodules https://github.com/belak/prezto-contrib contrib
```

Comment:
does this dir and env var exist by default in prezto?

Reply:
env var yes. `contrib` dir is the default for additional zprezto modules, lean theme was added there (see https://github.com/belak/prezto-contrib/tree/master/contrib-prompt/external)

Reply:
btw i should update lean ref there...

Reply:
ok, works for me